packages/wonder-stuff-ci/src/get-mobile-release-logger.ts

```diff
@@ -0,0 +1,79 @@
+import * as winston from "winston";
+import * as gcpLogger from "@google-cloud/logging-winston";
+const {combine, timestamp, json, errors, printf, colorize} = winston.format;
+
+/**
+ * Winston logger used for generating to GCP for the mobile release process (release-raccoon).
+ * @param {string} projectId The GCP project ID to send logs to.
+ * @param {string} logName The name of the log to send logs to in GCP.
+ * @param {string} logLevel The minimum severity level to log.
+ * @param {boolean} enableConsoleLogs If true, logs will be sent to the console as well as GCP.
+ * @param {[key: string]: string} labels K/V metadata that will be add to the logs
+ * @returns {winston.Logger} logger set up to send logs to GCP.
+ */
+export const getMobileReleaseLogger = (
+    args = {
+        projectId: "mobile-365917",
+        logName: "release-raccoon",
+        logLevel: "info",
+        enableConsoleLogs: false,
+        labels: {},
+    },
```

Comment:
To get the default values I would do this:
```suggestion
    {
        projectId = "mobile-365917",
        logName = "release-raccoon",
        logLevel = "info",
        enableConsoleLogs = false,
        labels = {},
    },
```

And then you would reference them by just saying `logLevel` instead of `args.logLevel`.

Reply:
Oh nice! Strange thing is in my IDE it complains if I pass nothing and just want the default values 
<img width="1063" alt="image" src="https://user-images.githubusercontent.com/34088613/231551847-f9ff806f-a65b-4b1b-a964-09db899d171f.png">

Reply:
Ah right - so you would have to have a *second* fallback in that case - you could provide that or just pass in an object object `{}` as the first argument.

Reply:
The easiest way to handle that, is this, I believe.
```suggestion
     {
        projectId = "mobile-365917",
        logName = "release-raccoon",
        logLevel = "info",
        enableConsoleLogs = false,
        labels = {},
     } = {},
```

If that doesn't work then you could do:

```
myFunc(options?: MyOptionsType) {
    const {
        projectId = "mobile-365917",
        logName = "release-raccoon",
        logLevel = "info",
        enableConsoleLogs = false,
        labels = {},
    } = options ?? {};
```

Reply:
I got it to work with the double fallback that John mentioned it looks like this.

```
export const getMobileReleaseLogger = (
    {
        redirectToStdout,
        labels,
    }: {
        redirectToStdout: boolean;
        labels: {[key: string]: string};
    } = {
        redirectToStdout: true,
        labels: {},
    },
): winston.Logger => {
```